exercises/clock/canonical-data.json

```diff
@@ -221,66 +221,66 @@
       "cases": [
         {
           "description": "subtract minutes",
-          "property": "add",
+          "property": "subtract",
           "hour": 10,
           "minute": 3,
-          "add": -3,
+          "subtract": 3,
```

Comment:
Since it matches the pattern for `add` I think it is OK.

Reply:
The `"property":` is  `"subtract"`, why does there have to be a value called `"subtract":` as well?  Since the README states that we are adding / subtracting minutes I think `"subtract":` and `"add":` should be called `"amount":` or `"value":`.